app/views/find/v2/results/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="app-filter-layout">
+  <div class="app-filter-layout__filter">
+    <div class="app-filter__header">
+      <h2 class="govuk-heading-m">Filters</h2>
+
+      <div class="app-filter__content">
+      </div>
+    </div>
+  </div>
+  <div class="app-filter-layout__content">
+    <ul class="app-search-results">
+      <% @results.each do |course| %>
+        <li class="app-search-results__item" data-qa="course">
+          <%= govuk_summary_card(
+            classes: ["course-summary-card"],
+            title: govuk_link_to(
+              find_course_path(
+                provider_code: course.provider_code,
+                course_code: course.course_code
+              ),
+              class: 'govuk-link govuk-!-font-size-24',
+              data: { qa: 'course__link' }
+            ) do
+              content_tag(:span, course.provider.provider_name, class: 'app-search-result__provider-name', data: { qa: 'course__provider_name' }) +
```

Comment:
On the original search results the code is calling smart_quotes method from publish helper. I added as checklist in the outcome card:

<img width="413" alt="Screenshot 2024-12-03 at 10 33 12" src="https://github.com/user-attachments/assets/9468c9ab-8e11-483b-be59-62f091828dc4">